CONTRIBUTING.md

```diff
@@ -1,11 +1,40 @@
 # How to Contribute
 
-TODO: docs on how to run the nextjs project and maybe even the fork/clone/pr flow
+1. Fork the repository
+2. Clone the repository to your local machine
+3. Create a new branch 
+```
+ git checkout -b <branch-name>
+```
 
-## Environment Variables
+4. Make your changes
+5. Commit and push your changes
+```
+ git add .
+ git commit -m "commit message"
+ git push origin <branch-name>
+```
+6. Create a pull request
+7. Wait for the pull request to be reviewed and merged
+
+
+# How to Setup Environment Variables
 
 Rename the file `.env.example` into `.env.local` and fill in the values.
 
-In order to make the login work you will need to create a new OAuth app on GitHub. You can find the instructions on the [Official GitHub Documentation](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app).
+## GitHub OAuth App
```

Comment:
I would probably mention somewhere that you don't need both OAuth and token, you just need one.

OAuth is the recommended way that matches production, but in development to make it easier we also accept the classic token.

Reply:
ok got it

Reply:
Even if the token is the quickest way, I think we should keep in the docs how to set up the App just in case someone wants to do it.

Reply:
yeah I will add